plugins/hydra_nevergrad_sweeper/example/dummy_training.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any
+
+import hydra
+
+log = logging.getLogger(__name__)
+
+
+@hydra.main(config_path="config.yaml")
+def dummy_training(cfg: Any) -> float:
+    """A dummy function to minimize
+    Minimum is 0.0 at:
+    lr = 0.12, dropout=0.33, db=mnist, batch_size=4
+    """
+    do = cfg.dropout
+    bs = cfg.batch_size
+    out = abs(do - 0.33) + int(cfg.db == "mnist") + abs(cfg.lr - 0.12) + abs(bs - 4)
+    logging.info(
+        "dummy_training(dropout=%s, lr=%s, db=%s, batch_size=%s) = %s",
+        do,
+        cfg.lr,
+        cfg.db,
+        bs,
+        out,
+    )
+    return out  # type: ignore
```

Comment:
Why type: ignore? is `out` not a float?

Reply:
it is but the typing did not detect it because the cfg object is not typed (right?)
I converted this to explicitely cast `out` at its creation, so that the comment is not needed anymore

Reply:
Oh yes.
Convert it to DictConfig type from Any. (it will not help but it's better).

I landed structured configs that can help here, but I don't think we should convert all examples.

When you have some time, take a look at the new tutorial and tell me what you think:
https://hydra.cc/docs/next/tutorials/structured_config/basic